engine/engine/content/builtins/scripts/mobdebug.lua

```diff
@@ -1278,9 +1308,11 @@ local function on()
   if co then
     coroutines[co] = true
     debug.sethook(co, debug_hook, HOOKMASK)
+    return true
   else
     if jit then coroutines.main = true end
     debug.sethook(debug_hook, HOOKMASK)
+    return true
```

Comment:
I added my hook under `on` functions, so I need to know if it was successfully set